tests/testthat/test-dm.R

```diff
@@ -76,58 +74,49 @@ test_that("'copy_to.dm()' works", {
     "need_unique_names"
   )
 
-  # copying local `tibble` to postgres `dm`
-  skip_if_error(
-    expect_equivalent_dm(
-      copy_to(dm_for_filter_src$postgres, d1_src$df, "test_table"),
-      dm_add_tbl(dm_for_filter_src$postgres, test_table = d1_src$postgres)
-    )
+  # copying `tibble` from chosen src to sqlite `dm`
+  expect_equivalent_dm(
+    copy_to(dm_for_filter_sqlite, d1, "test_table"),
+    dm_add_tbl(dm_for_filter_sqlite, test_table = d1_sqlite)
   )
 
-  # copying postgres `tibble` to local `dm`
-  skip_if_error(
-    expect_equivalent_dm(
-      copy_to(dm_for_filter_src$df, d1_src$postgres, "test_table_1"),
-      dm_add_tbl(dm_for_filter_src$df, test_table_1 = d1_src$df)
-    )
+  # copying sqlite `tibble` to `dm` on src of choice
+  expect_equivalent_dm(
+    copy_to(dm_for_filter, d1_sqlite, "test_table_1"),
+    dm_add_tbl(dm_for_filter, test_table_1 = d1)
   )
 })
 
 test_that("'compute.dm()' computes tables on DB", {
-  skip_if(is_empty(db_src_names))
-  walk(
-    db_src_names,
-    function(db_src_name) {
-      expect_true({
-        def <-
-          dm_for_filter_src[[db_src_name]] %>%
-          dm_filter(t1, a > 3) %>%
-          compute() %>%
-          dm_get_def()
-        test <- map_chr(map(def$data, sql_render), as.character)
-        all(map_lgl(test, ~ !grepl("WHERE", .)))
-      })
-    }
-  )
+  # FIXME: Regarding PR #313: should this be tested on object `dm_for_filter` and the test
+  # skipped if it is a local dm?
+  def <-
+    dm_for_filter_sqlite %>%
```

Comment:
`skip_if_local_src()` and

```suggestion
    dm_for_filter %>%
```